doc/source/user_guide/gotchas.rst

```diff
@@ -321,7 +321,7 @@ Byte-ordering issues
 --------------------
 Occasionally you may have to deal with data that were created on a machine with
 a different byte order than the one on which you are running Python. A common
-symptom of this issue is an error like:::
+symptom of this issue is an error like::
```

Comment:
Thanks @partev  - aren't there two extra colons though?

```suggestion
symptom of this issue is an error like:
```

Reply:
@MarcoGorelli the two colons are a restructuredtext construct to ensure the following block is a code block

Reply:
ah oops - thanks Joris!